cairo/src/image_surface.rs

```diff
@@ -149,6 +168,46 @@ impl ImageSurface {
     }
 }
 
+pub struct ImageSurfaceDataOwned {
```

Comment:
You need to export that from `lib.rs`: https://github.com/gtk-rs/gtk-rs-core/blob/master/cairo/src/lib.rs#L110

Reply:
Done